checks.py

```diff
@@ -67,7 +74,7 @@ def check_four_o_four(url):
             if h1:
                 result_dict['h1'] = h1.string
             else:
-                review = review + '* hittade ingen huvud rubrik (h1)\n'
+                review = review + _('TEST_404_REVIEW_MAIN_HEADER') #'* hittade ingen huvud rubrik (h1)\n'
```

Comment:
@flowertwig-org I cannot seem to get the Swedish translation to work, the English translation is used. In this case the terminal output is when using '-L sv':
"Omdöme:
 * Found no headline (h1)"

Any ideas?